msrestazure/azure_active_directory.py

```diff
@@ -559,11 +559,19 @@ def signed_session(self):
         return session
 
 
-def get_msi_token(resource, port=50342):
+def get_msi_token(resource, port=50342, client_id=None):
```

Comment:
FYI, msi's resource id and object id are also supported

Reply:
I saw your email about this naming. Will wait for official suggestion before merging.